content/docs/tasks/telemetry/export-logs-through-stackdriver.md

```diff
@@ -0,0 +1,301 @@
+              payloadTemplate: '{{or (.sourceIp) "-"}} - {{or (.sourceUser) "-"}} [{{or (.timestamp.Format "02/Jan/2006:15:04:05 -0700") "-"}}] "{{or (.method) "-"}} {{or (.url) "-"}} {{or (.protocol) "-"}}" {{or (.responseCode) "-"}} {{or (.responseSize) "-"}}'
+              httpMapping:
+                url: url
+                status: responseCode
+                requestSize: requestSize
+                responseSize: responseSize
+                latency: latency
+                localIp: sourceIp
+                remoteIp: destinationIp
+                method: method
+                userAgent: userAgent
+                referer: referer
+              labelNames:
+              - sourceIp
+              - destinationIp
+              - sourceService
+              - sourceUser
+              - sourceNamespace
+              - destinationIp
+              - destinationService
+              - destinationNamespace
+              - apiClaims
+              - apiKey
+              - protocol
+              - method
+              - url
+              - responseCode
+              - responseSize
+              - requestSize
+              - latency
+              - connectionMtls
+              - userAgent
+              - responseTimestamp
+              - receivedBytes
+              - sentBytes
+              - referer
+              sinkInfo:
+                id: '<sink_id>'
+                destination: '<sink_destination>'
+                filter: '<log_filter>'
+        ---
+        apiVersion: "config.istio.io/v1alpha2"
+        kind: rule
+        metadata:
+          name: stackdriver
+          namespace: istio-system
+        spec:
+          match: "true" # If omitted match is true.
+          actions:
+          - handler: handler.stackdriver
+            instances:
+            - requestcount.metric
+            - requestduration.metric
+            - requestsize.metric
+            - responsesize.metric
+            - accesslog.logentry
+        ---
+    ```
+
+1.  Push the configuration
+
+    ```command
+    $ kubectl apply -f stackdriver.yaml
+    stackdriver "handler" created
+    rule "stackdriver" created
+    logentry "stackdriverglobalmr" created
+    metric "stackdriverrequestcount" created
+    metric "stackdriverrequestduration" created
+    metric "stackdriverrequestsize" created
+    metric "stackdriverresponsesize" created
+    ```
+
+1.  Send traffic to the sample application.
+
+    For the Bookinfo sample, visit `http://$GATEWAY_URL/productpage` in your web
+    browser or issue the following command:
+
+    ```command
+    $ curl http://$GATEWAY_URL/productpage
+    ```
+
+1.  Verify that logs are flowing through Stackdriver and further to the sink
+    configured.
+
+    *   Stackdriver: Navigate to [Stackdriver Logs
+        Viewer](https://pantheon.corp.google.com/logs/viewer) for your project
+        and look under "GKE Container" -> "Cluster Name" -> "Namespace Id" for
```

Comment:
question: how will the MR work for logs impact this location?

Reply:
Not sure.. change is not checked-in yet.. will update once it's checked-in?